basic-data-structures/add-items-using-splice.js

```diff
@@ -0,0 +1,11 @@
+function htmlColorNames(arr) {
+  // Only change code below this line
+  const initialIndex = 0;
+  const NumberOfelementsExcluded = 2;
```

Comment:
Following the best practice of JS, you have to create variables starting with lowecase letters and use upper case if the name is compound of multiple words everytime you move from one word to another. So change this.